kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -1084,6 +1105,12 @@ class ResidualBasedNewtonRaphsonStrategy
      */
     bool mCalculateReactionsFlag;
 
+    /**
+     * @brief Flag telling if a full update of the database will be performed at the first iteration
+     * @details default = false
+     */
+    bool mUseOldStiffnessInFirstIteration = false;
```

Comment:
I would use a flag instead of a bool

Reply:
it is a question of taste...i don't see why not using a bool here (anyway we are not passing a flag from outside)